Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -105,10 +104,10 @@ add_decl_doc a₄
 /-- The `a₆` coefficient of a Weierstrass curve. -/
 add_decl_doc a₆
 
-instance instInhabitedWeierstrassCurve {R : Type u} [Inhabited R] :
+instance instInhabited {R : Type u} [Inhabited R] :
```

Comment:
Should these renames come with a `@[deprecated] alias`?

Reply:
My understanding is that explicit instance names do not get deprecation aliases. (That said: if you feel there should be consensus on it, feel free to further the zulip thread on a deprecation policy. I intend to return it it, but not in before three weeks.)